### PR TITLE
chore: add GraphQL query example on the app side

### PR DESCRIPTION
**Pull request by amille44420**
Add graphql-tag support in webpack

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,6 +58,7 @@
     "@thetribe/eslint-config-react": "^0.2.1",
     "babel-jest": "^24.9.0",
     "babel-loader": "8.0.6",
+    "babel-plugin-graphql-tag": "^2.5.0",
```
Comment (jvasseur) [inline package.json]: Is this needed since we load queries form external files and not from tags in the javascript source ?

Comment (amille44420) [reply package.json]: It avoid bundling `graphql-tag` and it directly inline its processing in our bundles.

Comment (amille44420) [reply package.json]: If we deny the use of tags we can remove it, but if someone does, without it we'll include `graphql-tag` in our bundle. I'm not against removing it

**Comment by amille44420** [comment]
@jvasseur I wanted to add `eslint-plugin-graphql`  but it means we have to keep a schema introspection in our versioning. What do you think about it ? The pro is we would ensure the queries files do match our schema.

**Comment by jvasseur** [comment]
@amille44420 I'm not found of committing the compiled schema, would work to do something like `schemaJson: require('./schema.js'),` with `schema.js` beeing a file that compile the schema and export it ?

File: package.json (hunk 1 above)
Comment (jvasseur) [reply package.json]: I generally prefer to use separate files for GraphQL queries so I wouldn't be against denying the use of the tag but I'm also ok for keeping it like this.

**Comment by amille44420** [comment]
@jvasseur do you think we can move on with this PR ? I need it for #182 and I believe we may add the GraphQL query linter at a late date. It's not an absolute priority for now.

**Comment by jvasseur** [comment]
I'm ok keeping it like this and adding the linter later.

**Comment by amille44420** [comment]
this feature will be shipped on `varaint/graphql` (#182)

**Pull request closed without merging**